clients/python/test/e2e/ci/_data_classes.py

```diff
@@ -0,0 +1,172 @@
+import configparser
+from pathlib import Path
+from typing import Dict, Optional
+from urllib.parse import ParseResult, urlparse
+
+from _utils import _PYTEST_INI
+from packaging.version import Version
+from pydantic import BaseModel, field_validator, model_validator
+
+# Holds classes for passing data around between scripts.
+
+
+class ServerConfig(BaseModel):
+    """Holds data about server configuration"""
+
+    osparc_api_host: str
+    osparc_api_key: str
+    osparc_api_secret: str
+
+    @property
+    def url(self) -> ParseResult:
+        return urlparse(f"{self.osparc_api_host}")
```

Comment:
is this already a URL? if yes I would suggest doing
```osparc_api_host: AnyUrl``` this would validate and then directly return a url.

Reply:
In fact I did this, but then when I convert to a string the url ends with a "/" and that causes issues. Hence, I decided to just keep it as a string.